auth-web/src/registerServiceWorker.ts

```diff
@@ -1,48 +1,15 @@
-/* eslint-disable no-console */
-
-import ConfigHelper from './util/config-helper'
-import { SessionStorageKeys } from './util/constants'
 import { register } from 'register-service-worker'
 
 if (process.env.NODE_ENV === 'production') {
   register(`${process.env.BASE_URL}service-worker.js`, {
-    ready () {
-      console.log(
-        'App is being served from cache by a service worker.\n' +
-        'For more details, visit https://goo.gl/AFskqB'
-      )
-    },
-    registered (event) {
-      console.log('Service worker has been registered.')
-    },
-    cached () {
-      console.log('Content has been cached for offline use.')
-    },
-    updatefound () {
-      console.log('New content is downloading.')
-    },
-    updated (updatedEvent) {
-      // write code for popup here
-      console.log('New content is available; refreshing page in 1 sec')
-      // showInteractiveToastReload('New content is available!', updatedEvent)
-
-      // Remove Launch Darkly files from session if new build content is downloaded
-      // Once the app is reloaded after the timeout, the App.vue will initialize the LD againx
-      // Otherwise, user will only get the LD flags if he/she signout-and-signin or restart-browser
```

Comment:
I think we still need these conditions, in the same function. Since we are using PWA, better to keep above conditions also